<ui version="4.0" >
 <class>QmitkLevelWindowRangeChange</class>
 <widget class="QDialog" name="QmitkLevelWindowRangeChange" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>110</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>210</width>
    <height>110</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>210</width>
    <height>110</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Change Scale Range</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="1" colspan="2" >
    <widget class="QSpinBox" name="rangeMaxSpinBox" >
     <property name="minimum" >
      <number>-9999999</number>
     </property>
     <property name="maximum" >
      <number>10000000</number>
     </property>
     <property name="value" >
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QPushButton" name="okButton" >
     <property name="text" >
      <string>OK</string>
     </property>
     <property name="autoDefault" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="lowerLimitLabel" >
     <property name="text" >
      <string>Lower Limit:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="upperLimitLabel" >
     <property name="text" >
      <string>Upper Limit:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QPushButton" name="cancelButton" >
     <property name="text" >
      <string>Cancel</string>
     </property>
     <property name="autoDefault" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="QSpinBox" name="rangeMinSpinBox" >
     <property name="minimum" >
      <number>-10000000</number>
     </property>
     <property name="maximum" >
      <number>9999999</number>
     </property>
     <property name="value" >
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>rangeMaxSpinBox</tabstop>
  <tabstop>rangeMinSpinBox</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <includes/>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>QmitkLevelWindowRangeChange</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>146</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
